game/games.go

```diff
@@ -1,62 +1,85 @@
 package game
 
 import (
+	"database/sql"
 	"encoding/json"
-	"errors"
 	"fmt"
+	"log"
 	"net/http"
 	"strconv"
 
 	"github.com/gorilla/mux"
+	_ "github.com/mattn/go-sqlite3"
 )
 
-type Games struct {
-	games []Game
-	seq   int
-}
+func findGame(db *sql.DB, id int64) (*Game, error) {
```

Comment:
If we stick with this pattern we should move this to `game`

Reply:
feels like we can probably migrate everything to `games` and get ride of `game`